server.js

```diff
@@ -30,7 +30,7 @@ app.use(express.static('public'));
 //this is rendering the entire index page
 
 app.get('/', (reg, res) => {
-  res.render('pages/index');
+  res.render('pages/index',{books: fetchBooksFromDB(req, res)});
```

Comment:
this line of code attempts to evaluate asynchronously and therefore nothing goes to your index
Instead you need to call 
```
fetchBooksFromDB()
.then(books => {
   res.render('pages/index', {books: books})
})
```